src/app/core/components/header/header.component.html

```diff
@@ -15,18 +15,25 @@
 
         <ul class="navigation-main__list">
           <li class="navigation-main__item">
-            <a [routerLink]="['/about']" class="navigation-main__link" title="Über uns">Über uns</a>
+            <a href="javascript:;" title="Fall einreichen"
```

Comment:
What is the `href="javascript:;"` necessary for?

Reply:
I don't know. It has been there before, so I didn't really look at it. Removed it now and everything seems to still work fine.